lichess.py

```diff
@@ -153,3 +153,8 @@ def challenge(self, username, params):
     def cancel(self, challenge_id):
         return self.api_post(ENDPOINTS["cancel"].format(challenge_id),
                              raise_for_status=False)
+
+    def online_book_get(self, path, params=None):
+        response = self.session.get(path, timeout=2, params=params)
+        response.raise_for_status()
```

Comment:
`response.raise_for_status()` will raise an error when lichess cloud eval doesn't have a move. This is why the `raise_for_status` parameter was added to `api_get`. I think in this case you can remove it entirely instead of adding a parameter.

Reply:
`raise_for_status` and `params` can also be removed from `api_get`.

Reply:
The `challenge()` and `cancel()` methods have `raise_for_status=False`, so I don't think that parameter can be removed from `api_get()`. Several methods have `params` like `make_move()`, so I don't think this can be removed either.

Reply:
`challenge`, `cancel` and `make_move` use `api_post` and not `api_get`, so they can be removed from `api_get`.

Reply:
I see. Parameters removed. `get_game_pgn()` used `params`, but that didn't actually do anything.

I have a feeling that the `data` and `payload` parameters of `api_post()` are redundant, but that's for another PR.